kpops/components/streams_bootstrap_v3/producer/producer_app.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from functools import cached_property
+
+from pydantic import Field, computed_field
+from typing_extensions import override
+
+from kpops.components.base_components.kafka_app import (
+    KafkaAppCleaner,
+)
+from kpops.components.common.streams_bootstrap import StreamsBootstrap
+from kpops.components.common.topic import (
+    KafkaTopic,
+    OutputTopicTypes,
+    TopicConfig,
+)
+from kpops.components.streams_bootstrap_v3.app_type import AppType
+from kpops.components.streams_bootstrap_v3.kafka_app import KafkaAppV3
+from kpops.components.streams_bootstrap_v3.producer.model import ProducerAppValues
+from kpops.utils.docstring import describe_attr
+
+log = logging.getLogger("ProducerApp")
+STREAMS_BOOTSTRAP_V3 = "3.0.0"
```

Comment:
3.0.0 sadly won't happen

Reply:
Oh no! Why? what would be the version then?

Reply:
We will need to discuss the version internally